### PR TITLE
Adding the DefaultUILanguage in appsettings.json file

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 10/umbraco-cms/tutorials/multilanguage-setup.md [hunk 1]
```diff
@@ -51,10 +51,28 @@ To change the default language of a website:
...
+        "Global": {
+            "DefaultUILanguage": "fr"
+            }
+          }
+        }
+    }
+
+    ```
+
+7. Save the `appsettings.json` file.
```
Comment (jonat123) [inline 10/umbraco-cms/tutorials/multilanguage-setup.md:75]: To be honest I don't think this is necessary to have here 🤔

This section is just about adding multiple languages in the settings section, which is used for having different languages displayed on the frontend.

However, the DefaultUILanguage is for changing the language used in the backoffice, so adding that setting should not be relevant in this context 😊

I hope it makes sense!

Comment (eshanrnh) [reply 10/umbraco-cms/tutorials/multilanguage-setup.md:75]: @jonat123 I agree, it's slightly a misfit for this article. 🙈

Would it make sense to add the info in the [Language Variants](https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/variants) article as a section at the end? 🤔

I think its important to have this info somewhere in the docs, though..

Comment (jonat123) [reply 10/umbraco-cms/tutorials/multilanguage-setup.md:75]: I think the problem is that this is not related to Language variants 😊

Also this is already documented here: https://docs.umbraco.com/umbraco-cms/reference/configuration/globalsettings#default-ui-language :-)

Comment (eshanrnh) [reply 10/umbraco-cms/tutorials/multilanguage-setup.md:75]: Ummm okay.. If it's not needed, I'll close this PR then 💪

**Pull request closed without merging**